### PR TITLE
PEP8 fixes to code base

### PR DESCRIPTION
**Pull request by kshitij10496**
I find this ironic.

### CODE REVIEW

**Comment by OrkoHunter** [comment]
I was waiting for the bot to fix its own code. :)

**Comment by kshitij10496** [comment]
> I was waiting for the bot to fix its own code. :)

That would be a good idea. However, we might not prefer PEP8 compliance everywhere possible.

File: app.py [hunk 1]
```diff
@@ -154,8 +158,8 @@ def main():
                     data["results"][file] = f.readlines()
                 data["results"][file] = [i.replace("file_to_check.py", file)[1:] for i in data["results"][file]]
 
-                ## Remove the errors and warnings to be ignored from config
-                ## Also remove other errors in case of diff_only = True
+                # Remove the errors and warnings to be ignored from config
+                # Also remove other errors in case of diff_only = True
```
Comment (OrkoHunter) [inline app.py:163]: Please don't convert `##` to `#`. If you notice, I'm using it as headers like markdown.

Comment (kshitij10496) [reply app.py:163]: Gotcha ! 
I was wondering why did you use double hashes as comments. 
Nice idea. 👍

**Comment by OrkoHunter** [comment]
@kshitij10496 Added `.pep8speaks.yml` to the repository with some errors to ignore. I don't prefer them for this project.

Now, did you try to run the app locally? The change at L218 will crash the app. `\` is not needed if the concatenation is withing braces, I think. So, we need to keep `\` there.

**Comment by kshitij10496** [comment]
> Now, did you try to run the app locally? The change at L218 will crash the app. \ is not needed if the concatenation is withing braces, I think. So, we need to keep \ there.

I agree with you. I have fixed it now.

**Comment by OrkoHunter** [comment]
Cool thanks!

**Pull request merged**